assets/alertmanager/alertmanager.yaml

```diff
@@ -27,6 +27,15 @@ spec:
         topologyKey: kubernetes.io/hostname
   automountServiceAccountToken: true
   containers:
+  - env:
+    - name: HTTP_PROXY
```

Comment:
just a tiny minor thing, I was wondering if we should also pass lower cased versions of these envs as well (http_proxy, https_proxy, no_proxy)